windows_docker/src/win_docker.py

```diff
@@ -0,0 +1,120 @@
+from checks import AgentCheck
+from datetime import tzinfo, timedelta
+import datetime
+import requests
+import json
+import calendar
+
+class UTC(tzinfo):
+  def utcoffset(self, dt):
+    return timedelta(0)
+  def tzname(self, dt):
+    return "UTC"
+  def dst(self, dt):
+    return timedelta(0)
+
+class WinDocker(AgentCheck):
+	def check(self, instance):
+		container_full_ids = []
+		container_ids = []
+		container_urls = []
+		id_tag = ""
+		image_tag = ""
+		state_tag = ""
+		status_tag = ""
+		name_tag = ""
+		net_tmpl = "docker.windows.networks.{}"
+		blkio_stats_tmpl = "docker.windows.blkio_stats.{}"
+		storage_stats_tmpl = "docker.windows.storage_stats.{}"
+		cpu_stats_tmpl = "docker.windows.cpu_stats.cpu_usage.{}"
+		throttling_data_tmpl = "docker.windows.cpu_stats.throttling_data.{}"
+		precpu_stats_cpu_usage_tmpl = "docker.windows.precpu_stats.cpu_usage.{}"
+		precpu_stats_throttling_data_tmpl = "docker.windows.precpu_stats.throttling_data.{}"
+		memory_stats_tmpl = "docker.windows.memory_stats.{}"
+
+		container_url="http://localhost:2375/containers/json"
+		containers_response = requests.get(container_url, timeout=1).json()
+
+
+		for value in containers_response:
+			container_full_ids.append(value["Id"])
+			container_count = len(containers_response)
+			
+			if value:
+				id_tag = "sources:docker_win"
+				image_tag = "docker_image:" + value["Image"]
```

Comment:
is the Image field in this json always fully resolved? Sometimes in docker inspect we see things like `"datadog/agent@sha256:5fbe2c87d4c4c2d9fc2d4bf45e39b0572f7f3f8ff906e3a77515a3308ab0c2cf"`, in which case we need to resolve the image tags, see https://github.com/DataDog/dd-agent/blob/master/utils/dockerutil.py#L571-L615

Reply:
Also could you follow the convention? docker_image is the full image name, image_name is the name without the tag, image_tag is the opposite, and short_image is the image_name without the registry, if there is one. See https://github.com/DataDog/datadog-agent/blob/master/pkg/util/containers/image.go#L13-L45

Reply:
Still looking into how to achieve this.